examples/example-hono-api/README.md

```diff
@@ -0,0 +1,39 @@
+# Fastify Example
+
+This example demonstrates how to use the `auth0-api-js` package to protect API's in a Hono application.
+
+## Install dependencies
+
+Install the dependencies using npm:
+
+```bash
+npm install
+```
+
+## Configuration
+
+```bash
+npx wrangler secret put AUTH0_DOMAIN
+npx wrangler secret put AUTH0_AUDIENCE 
+```
+
+Update types (optional)
+```bash
+npm run cf-typegen
+```
+
+With the configuration in place, the example can be deployed by running:
+
+```bash
+npm run deploy
+``` 
+
+## Endpoints
+
+The example API has the following endpoints:
+
+- `GET /api/public`: A public endpoint that can be accessed without authentication.
+- `GET /api/private`: A private endpoint that can only be accessed by authenticated users.
+- `GET /api/private/scope`: A private endpoint that can only be accessed by authenticated users with the `read:data` scope.
```

Comment:
nitpicking here, but can we use the same scope in all examples? The other samples are using `read:private`. If we do not like that scope, we can rename the others to `read:data` (I can do that, no need to do that in this PR, but curious to your thoughts?)